<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBusDialog</class>
 <widget class="QDialog" name="addBusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add bus</string>
  </property>
  <widget class="QDialogButtonBox" name="addBusButtonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>411</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_20">
    <item>
     <widget class="QLabel" name="label_55">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_56">
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="licnr_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_57">
      <property name="text">
       <string>Company Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="comname_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_58">
      <property name="text">
       <string>Organization Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="orgnr_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_59">
      <property name="text">
       <string>Bus type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bustyp"/>
    </item>
    <item>
     <widget class="QLabel" name="label_60">
      <property name="text">
       <string>Inside Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="insnr">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_62">
      <property name="text">
       <string>Number of seats:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="nrofsea">
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addBusButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addBusDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBusButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addBusDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>licnr_B</sender>
   <signal>textEdited(QString)</signal>
   <receiver>addBusButtonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
